### ISSUE TITLE
HPND: Modify template or create new license variant

### ISSUE THREAD

**Issue by swinslow** [spdx/license-list-XML#753]
[HPND](https://spdx.org/licenses/HPND.html) is a heavily templated license, matching to many slight variations in the wild.

One variation that was used for MIT Kerberos is almost a match for HPND's template, but with the following differences:

1. it includes "and sell" in the list of grants at the beginning
2. it moves the "without fee" language slightly
3. it uses "appears" instead of "appear"

Examples of this can be seen:

- in the Linux kernel: e.g. https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/net/sunrpc/auth_gss/gss_generic_token.c?h=v4.19
- elsewhere in the wild: https://support.brother.com/g/s/id/htmldoc/mfc/dcp_mfc9010cn/gb/html/uk/nug/appendix13_1_3.html (search that page for "OpenVision")

I'd be interested in thoughts on whether HPND's template should be extended to make HPND match these licenses, or whether these should be different licenses. 

My view is that (2) and (3) above are clearly non-substantive changes. But I'm wondering whether the addition of "and sell" in (1) is enough that these should be treated as a different license for SPDX matching purposes.

Either way, I can submit a PR to either extend the HPND template, or else to submit this as a new license. Just wanted to get the group's thoughts first before preparing a change for review.

**Comment by pombredanne** [comment]
@swinslow FWIW this is a variant that been tracked as its own license in ScanCode for quite a while as `x11-keith-packard` 
See https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/x11-keith-packard.LICENSE

**Comment by jlovejoy** [comment]
discussed on Feb 7 call - decided to add as new license. @swinslow to add new license issue for that
name? HPND-sell-variant ?
is variant here - https://fedoraproject.org/wiki/Licensing:MIT#Old_Style

**Comment by swinslow** [comment]
@jlovejoy I reviewed the variants on the Fedora page, and two of them have the similar "and sell" part of the license grant in the initial sentence. Those are:

- https://fedoraproject.org/wiki/Licensing:MIT#Old_Style
- https://fedoraproject.org/wiki/Licensing:MIT#Hylafax_Variant

But neither one has the same HPND-style disclaimer language that's found the variant I linked to above. So I'm going ahead and preparing a new PR with markup for this particular instance.
